code-studio/src/js/components/BeeCellEditor.jsx

```diff
@@ -0,0 +1,78 @@
+/* global React */
+
+var BeeCell = require('blockly-mooc/src/maze/beeCell');
+
+module.exports = React.createClass({
+  propTypes: {
+    cell: React.PropTypes.object.isRequired,
```

Comment:
it has half a dozen parameters, all but one of which are optional. I figured it was best to leave it as is.